dev.js

```diff
@@ -0,0 +1,28 @@
+const Bundler = require('parcel-bundler');
+const Path = require('path');
+const util = require('util');
+const exec = util.promisify(require('child_process').exec);
```

Comment:
i would add vertical spacing between this line and line 3. I kinda get that this is a _special_ `require`, but since this depends on the fact that a _standard_ requirement has been called before, it could break if someone changes the order of the requirements.

at the same time, since line 4 would be vertically spaced from _standard_ requirements, you will be able to sort alphabetically lines 1 to 3.